test/visual/default.html

```diff
@@ -4,7 +4,10 @@
   <meta charset="UTF-8">
   <title></title>
   <script src="../../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
-  <link href="../../vaadin-element.html" rel="import">
+  <script>
+    const theme = window.location.search.replace(/.*theme=(\w+).*/, '$1') || 'lumo';
+    document.write('<link rel="import" href="../../theme/' + theme + '/vaadin-element.html">');
```

Comment:
Wow, `document.write` in 2018. That's cool :)

Consider using the less cool and more predictable/maintainable solution: https://stackoverflow.com/a/577002/1331425